definitions/npm/cypress_v5.x.x/flow_v0.25.x-/cypress_v5.x.x.js

```diff
@@ -0,0 +1,803 @@
+declare type Cypress$Encoding =
+  | 'ascii'
+  | 'base64'
+  | 'binary'
+  | 'hex'
+  | 'latin1'
+  | 'utf8'
+  | 'utf-8'
+  | 'ucs2'
+  | 'ucs-2'
+  | 'utf16le'
+  | 'utf-16le';
+
+declare type Cypress$FileContents = string | Array<any> | { ... };
+
+declare type Cypress$HistoryDirection = 'back' | 'forward';
+
+declare type Cypress$HttpMethod =
+  | 'GET'
+  | 'POST'
+  | 'PUT'
+  | 'DELETE'
+  | 'OPTIONS'
+  | 'HEAD'
+  | 'TRACE'
+  | 'CONNECT';
+
+declare type Cypress$ViewportPreset =
+  | 'ipad-2'
+  | 'ipad-mini'
+  | 'iphone-3'
+  | 'iphone-4'
+  | 'iphone-5'
+  | 'iphone-6'
+  | 'iphone-6+'
+  | 'iphone-x'
+  | 'iphone-xr'
+  | 'macbook-11'
+  | 'macbook-13'
+  | 'macbook-15'
+  | 'samsung-note9'
+  | 'samsung-s10';
+
+declare type Cypress$RequestBody = string | { ... };
+
+declare type Cypress$RequestResponse = {
+  status: any,
+  body: any,
+  headers: any,
+  duration: any,
+  ...
+};
+
+declare type Cypress$ViewportOrientation = 'portrait' | 'landscape';
+
+declare type Cypress$Env = { [key: string]: string, ... };
+
+declare type Cypress$Window = any;
+
+declare interface Cypress$Core {
+  /**
+   *
+   * @see https://docs.cypress.io/api/cypress-api/custom-commands.html
+   */
+  addChildCommand(name: string, fn: Function): void,
```

Comment:
`Function` has been aliased to `any` from Flow 0.88.0.

**suggestion:** replace with a real function type?

Reply:
Wow! I didn't know that, this is so misleading. I found the docs for it so I'll make the changes to use `(...args: Array<any>) => any` instead

https://flow.org/en/docs/types/functions/#toc-function-type